hazelcast/src/main/java/com/hazelcast/internal/metrics/impl/DoubleGaugeImpl.java

```diff
@@ -61,7 +61,11 @@ public double read() {
     }
 
     @Override
-    public void render(StringBuilder stringBuilder) {
-        stringBuilder.append(read());
+    public void render(StringBuilder stringBuilder, String unboundValue) {
+        if (isBound()) {
```

Comment:
should we call getProbeInstance() before isBound check to set the probeInstance (or it can be done from within isBound call).